tensorboard/plugins/histogram/summary_test.py

```diff
@@ -175,6 +175,24 @@ class SummaryV2PbTest(SummaryBaseTest, tf.test.TestCase):
     def histogram(self, *args, **kwargs):
         return summary.histogram_pb(*args, **kwargs)
 
+    def test_singleton_input(self):
```

Comment:
Can we add the other 3 test cases for v3 as well?

test_empty_input, test_empty_input_of_high_rank, test_zero_bucket_count

See also other comment about the empty input case.

Reply:
Done.